step-framework-parent/step-framework-collections/src/test/java/step/core/accessors/CachedAccessorTest.java

```diff
@@ -16,6 +17,7 @@ public void before() {
 		accessor = new CachedAccessor<AbstractIdentifiableObject>(underlyingAccessor);
 		organizableObjectAccessor = new CachedAccessor<AbstractOrganizableObject>(underlyingOrganisableObjectAccessor);
 		beanAccessor = new CachedAccessor<Bean>(underlyingBeanAccessor);
+		underlyingBeanAccessor.enableVersioning(new InMemoryCollection<>(), 1l);

```

Comment:
Not sure if it makes sense to run the tests with and without versioning to be sure that it still continue to works when the versioning is disabled (no NPE , etc)

Reply:
I modified the tests to separate the versioning tests and execute with and without versioning